src/assets/js/fluent.js

```diff
@@ -0,0 +1,79 @@
+import { FluentBundle, FluentResource } from '@fluent/bundle';
+import insane from 'insane';
+import Strings from '@/locales/en-US/strings.ftl';
+
+class Fluent {
+  constructor(locale = 'en-US') {
+    const resource = new FluentResource(Strings);
+    const bundle = new FluentBundle(locale);
+    const errors = bundle.addResource(resource);
+    if (errors.length) {
+      console.error(errors);
+    }
+
+    this.bundle = bundle;
+  }
+
+  get(...parameters) {
+    if (typeof parameters[0] === 'string') {
+      switch (parameters.length) {
+        case 2:
+          if (typeof parameters[1] === 'string') {
+            return this.get({ id: parameters[0], attr: parameters[1] });
+          }
+          return this.get({ id: parameters[0], args: parameters[1] });
+        default:
+          return this.get({ id: parameters[0] });
+      }
+    }
+    const { id, attr, args, tags } = parameters[0];
+    const message = this.getMessage(id, attr, args);
+    return Fluent.sanitize(message, tags);
+  }
+
+  static sanitize(message, tags = {}) {
+    return insane(message, {
+      filter(token) {
+        const name = token.attrs['data-l10n-name'];
+        if (Object.keys(tags).includes(name) && tags[name].tag === token.tag) {
+          Object.keys(token.attrs).forEach((k) => {
+            if (k !== 'title') delete token.attrs[k];
```

Comment:
Agreed, I thought the linter would have caught this, but apparently not - I've removed this section of code anyway because of some bugs I found while writing tests